TestFoundation/TestNSString.swift

```diff
@@ -1385,6 +1405,10 @@ func checkHasPrefixHasSuffix(_ lhs: String, _ rhs: String, _ stack: [UInt]) -> I
 extension TestNSString {
     func test_PrefixSuffix() {
         for test in comparisonTests {
+            if case .skip = test.behavior {
```

Comment:
the PR is claiming a skip of a single test but that seems rather contrary to the additions and refactoring of reasons to xfail etc. Can we split this up into two parts?

Reply:
Normally, I'd be fine with that but we're trying to unblock testing and I don't think it's worth while to have 2 PRs for such a small change. I can split it up if you insist but I'd rather just unblock CI.